app/api/scraping/tiktok/route.ts

```
@@ -52,114 +41,122 @@ interface ScrapeCreatorsResponse {
+                reason: 'invalid_keywords',
+                keywords 
+            }, LogCategory.TIKTOK);
+            return createErrorResponse('Keywords are required and must be an array', 400, requestId);
         }
 
-        console.log('🔍 Paso 5: Sanitizando keywords');
-        // Sanitizar las keywords para evitar problemas de codificación
+        // Sanitize keywords
         const sanitizedKeywords = keywords.map(keyword => {
-            // Eliminar caracteres no válidos de UTF-8
             const sanitized = keyword.replace(/[\u0000-\u001F\u007F-\u009F\uD800-\uDFFF]/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)